vpr/test/test_vpr_constraints.cpp

```diff
@@ -425,6 +426,34 @@ TEST_CASE("RegionLocked", "[vpr]") {
     REQUIRE(is_r2_locked == false);
 }
 
+//Test calculation of macro constraints
+TEST_CASE("MacroConstraints", "[vpr]") {
+    t_pl_macro pl_macro;
```

Comment:
What is pl_macro initialized to?  Maybe you're relying on the default constructor?  Better to initialize it I think, and describe what the test does (macro of size 1, macro of size 2, ??).

Reply:
The pl_macro wasn't initialized to anything. It is only passed in to update_macro_member_pr so that if there is an error the error message can be printed with the macro. This won't happen in the case of this unit test, since the test is just checking that the PartRegion is updated properly for a particular case. I can initialize it to something though, and yes, I'll add some comments about what the test is doing.